sepal_ui/sepalwidgets/inputs.py

```diff
@@ -10,6 +10,7 @@
 from natsort import humansorted
 from traitlets import Any, Bool, Dict, Int, List, Unicode, link, observe
 
+import sepal_ui.sepalwidgets as sw
```

Comment:
As we are living in the sepalwidgets module you cannot yet import it inside one of its files (as __ini__.py is also importing inputs and all the others). In short, that's a circular reference and it can lead to an undebugable situation (that we already face at the beginning). 
Could you fall back to the previous implementation (with v.Layout and SepalWidget class) ?